go.mod

```diff
@@ -31,6 +31,7 @@ require (
 	github.com/spf13/pflag v1.0.5
 	github.com/spf13/viper v1.19.0
 	github.com/stretchr/testify v1.9.0
+	github.com/test-go/testify v1.1.4
```

Comment:
Up until now we managed to do our mocks and overall testing without testify. Shall we bring it now? It will create some discrepancies I think.

Reply:
Pardon, I did not catch we already have `github.com/stretchr/testify`... fixing now.